src/transaction/boot_sr.c

```diff
@@ -2229,6 +2229,15 @@ boot_restart_server (THREAD_ENTRY * thread_p, bool print_restart, const char *db
 
   init_server_type (db_name);
 
+  if (get_server_type () == SERVER_TYPE_PAGE && !HA_DISABLED ())
+    {
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_PRM_CONFLICT_EXISTS_ON_MULTIPLE_SECTIONS, 6, "cubrid.conf", "common",
```

Comment:
ER_PRM_CONFLICT_EXISTS_ON_MULTIPLE_SECTIONS is not a perfect match for the page server & HA situation. It is preferable to have a new error message.